apple-nav/src/Data.js

```diff
@@ -0,0 +1,109 @@
+const Data = {
```

Comment:
Wow, you weren't kidding about this taking the longest time to complete. It's really impressive.